docs/src/Contributing.md

```diff
@@ -70,7 +70,35 @@ in `docs/`.
 When your PR is ready for review, clean up your commit history by squashing
 and make sure your code is current with `ClimateMachine` master by rebasing.
 
-### Squash and rebase
+## Reformatting your code with `.dev/climaformat.jl`
+
+Your PR cannot be merged unless it conforms to the format style specified
+by the formatter implemented in the script `.dev/climaformat.jl`.
+
+To use the script `.dev/climaformat.jl` to reformat every file in
+`ClimateMachine.jl/` run
+
+```
+julia --project .dev/climaformat.jl
+```
+
+from `ClimateMachine.jl`'s root directory. To format individual files, type
+
+```
+julia --project .dev/climaformat.jl src/individual_file.jl
+```
+
+instead.
+
+!!! warn "Reformatting edits your code"
+    If you would like to be able to revert changes made by the formatter,
+    1. Create a commit that contains your unformatted contributions before running
+       the formatter.
```

Comment:
Technically it is sufficient to add them to the index (i.e. have been `git add`ed). This is probably advisable anyway, since it will only format files that are in the index (so if you create a new .jl file, the formatter won't be applied until you `git add` it).